scripts/portstat

```diff
@@ -83,24 +84,33 @@ class Portstat(object):
         # Build a dictionary of the stats
         cnstat_dict = OrderedDict()
         cnstat_dict['time'] = datetime.datetime.now()
+        if counter_port_name_map is None:
+            return cnstat_dict
         for port in natsorted(counter_port_name_map):
             cnstat_dict[port] = get_counters(counter_port_name_map[port]) 
         return cnstat_dict
 
-    def get_port_status(self, port_name):
+    def get_port_oper_status(self, port_name):
```

Comment:
you could combine the `get_port_oper_status` and `get_port_admin_status` functions into one.